ospfd/ospf_opaque.h

```diff
@@ -77,6 +77,46 @@
 	((ntohs((lsahdr)->length) >= sizeof(struct lsa_header))                \
 	 && ((ntohs((lsahdr)->length) % sizeof(u_int32_t)) == 0))
 
+/*
+ * Following section defines generic TLV (type, length, value) macros,
+ * used for various LSA opaque usage e.g. Traffic Engineering.
+ */
+struct tlv_header {
+  u_int16_t     type;                   /* Type of Value */
```

Comment:
Correct Linux kernel style errors raised by the script. But, how to for Warning ? All of them are about "WARNING: struct  should normally be const" but I'm pretty sure that this not apply.

Ah and using same script on ospfd.h (which I don't modify in this path) give me total: 27 errors, 224 warnings, 571 lines checked :-D